### ISSUE TITLE
can't resolve file

### ISSUE THREAD

**Issue by keefer223** [ArduPilot/ChibiOS#91]
/modules/ChibiOS/ext/fatfs/documents/00index_e.html

line 9     <link rel="up" title="Freewares" href="../../fsw_e.html

I couldn't get ardupilot to run.

Debian 12, python 3.11,

so I loaded ardupilot into pycharm. Unfortunately, pycharm couldn't find fsw_e.html.
I even did a search of ardupilot and was still not able to find fsw_e.html,

**Comment by khancyr** [comment]
this is a just a chibios os documentation file. it doesn't prevent anything. Moreover, ardupilot is using c++ not python so pycharm is the wrong IDE to try to do something.
Please use the wiki to learn how to compile the autopilot.

**Comment by keefer223** [comment]
thanks, I am aware of the c++ files. As soon as I finish with the python
cleanup I'll transition to Clion.

Some of the python and css files are still using some outdated code. I was
surprised to see some python2.X code.

The only reason I jumped onto IDE's was because I spent several days trying
to get ArduPilot up and running; first on a laptop w/ debian, then on a
dual processor server.

I wanted to get the software up and running before I connected the
agriculture hexacopter. wing span: 975mm.

keith
